### PR TITLE
Add media control for local stream

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/controls/MediaControl.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+export interface IMediaControlProps {
+    onToggleMute: (event: any) => void;
+    onToggleDisable: (event: any) => void;
+}
+
+export interface IMediaControlState { }
+
+export class MediaControl extends React.PureComponent<IMediaControlProps, IMediaControlState> {
```
Comment (asinner) [inline src/components/controls/MediaControl.tsx]: `MediaControl` => `MediaStreamControls`

File: src/components/controls/MediaControl.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+export interface IMediaControlProps {
```
Comment (asinner) [inline src/components/controls/MediaControl.tsx]: add `stream: MediaStream`

File: src/components/Conference.tsx [hunk 3]
```diff
@@ -282,4 +287,20 @@ export class Conference extends React.Component<IConferenceProps, {}> {
     private getPeerConnectionById(id: string) {
         return this.peerConnections[id]
     }
+
+    private onToggleLocalMute(event: any) {
```
Comment (asinner) [inline src/components/Conference.tsx]: Move this inside `MediaControl`

File: src/components/Conference.tsx [hunk 4]
```diff
@@ -282,4 +287,20 @@ export class Conference extends React.Component<IConferenceProps, {}> {
...
+        if (!this.localStream) {
+            console.warn('No local stream');
+            return;
+        }
+        this.localStream.getAudioTracks()[0].enabled = !this.localStream.getAudioTracks()[0].enabled
+    }
+
+    private onToggleLocalDisable(event: any) {
```
Comment (asinner) [inline src/components/Conference.tsx]: Move this inside `MediaControl`

File: src/components/controls/MediaControl.tsx (hunk 2 above)
Comment (niuyunsi) [reply src/components/controls/MediaControl.tsx]: Done

File: src/components/controls/MediaControl.tsx (hunk 1 above)
Comment (niuyunsi) [reply src/components/controls/MediaControl.tsx]: Done

File: src/components/Conference.tsx (hunk 3 above)
Comment (niuyunsi) [reply src/components/Conference.tsx]: Done

File: src/components/Conference.tsx (hunk 4 above)
Comment (niuyunsi) [reply src/components/Conference.tsx]: Done

**Pull request merged**